diff --git a/sqlite3.c b/sqlite3.c
index 4ec1271..e05ad74 100644
--- a/sqlite3.c
+++ b/sqlite3.c
@@ -51291,7 +51291,8 @@ void enable_simulated_io_errors(void){
 ** the error code is returned to the caller and the contents of the
 ** output buffer undefined.
 */
-SQLITE_PRIVATE int sqlite3PagerReadFileheader(Pager *pPager, int N, unsigned char *pDest){
+extern int sqlite3PagerReadFileheader(Pager *pPager, int N, unsigned char *pDest);
+SQLITE_PRIVATE int _sqlite3PagerReadFileheader(Pager *pPager, int N, unsigned char *pDest){
   int rc = SQLITE_OK;
   memset(pDest, 0, N);
   assert( isOpen(pPager->fd) || pPager->tempFile );
